common/src/main/java/whocraft/tardis_refined/common/block/device/ConsoleConfigurationBlock.java

```diff
@@ -122,6 +122,34 @@ public InteractionResult use(BlockState blockState, Level level, BlockPos blockP
                 }
                 return InteractionResult.CONSUME;
             } else {
+
+                if (player.isCrouching()) {
+                    if (level.getBlockEntity(blockPos.offset(offset)) instanceof GlobalConsoleBlockEntity globalConsoleBlock) {
+                        globalConsoleBlock.killControls();
+                        level.destroyBlock(blockPos.offset(offset), true);
+                    }
+
+                } else {
+                    level.setBlockAndUpdate(blockPos, blockState.setValue(ConsoleConfigurationBlock.CONSOLE, nextTheme));
+                    if ((consoleBlock.getBlock() instanceof GlobalConsoleBlock)) {
+                        level.setBlockAndUpdate(blockPos.offset(offset), BlockRegistry.GLOBAL_CONSOLE_BLOCK.get().defaultBlockState().setValue(GlobalConsoleBlock.CONSOLE, nextTheme));
+                        if (Platform.isClient()) {
+                            level.playSound(null, blockPos.offset(offset), SoundEvents.ENCHANTMENT_TABLE_USE, SoundSource.BLOCKS, 3, 0.45f);
+                            int i;
+                            double d;
+                            double e;
```

Comment:
Messy particle code. Use meaningful names and consider creating a helper function for spawning particles.